test/tests/varnish/run.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+set -eo pipefail
+
+set -x
```

Comment:
Some debugging got left in. :eyes: 

Everything else looks good to me :+1:   Thanks!

Reply:
that one was actually intentional, I swear! I figured it wouldn't harm to have verbose/noisy that would save time if something went wrong in the future. Is there a rule about this? If so, I might have to silence the `sidecar` commands as they are bound to be chatty

Reply:
in doubt, I silenced everyone

Reply:
Sorry for the delay. Yeah, the tests are designed to only be noisy on failure. Thanks for updating!